pkg_resources/__init__.py

```diff
@@ -557,7 +558,7 @@ def metadata_isdir(self, name: str) -> bool:
         """Is the named metadata a directory?  (like ``os.path.isdir()``)"""
 
     def metadata_listdir(self, name: str):
-        """List of metadata names in the directory (like ``os.listdir()``)"""
+        """list of metadata names in the directory (like ``os.listdir()``)"""
```

Comment:
This one and the next are a bit unexpected 😅

Reply:
```suggestion
        """List of metadata names in the directory (like ``os.listdir()``)"""
```